lighthouse-core/gather/connections/cri.js

```diff
@@ -10,19 +10,18 @@ const WebSocket = require('ws');
 const http = require('http');
 const log = require('lighthouse-logger');
 
-const hostname = 'localhost';
+const DEFAULT_HOSTNAME = 'localhost';
 const CONNECT_TIMEOUT = 10000;
 const DEFAULT_PORT = 9222;
 
 class CriConnection extends Connection {
   /**
    * @param {number=} port Optional port number. Defaults to 9222;
+   * @param {string=} hostname Optional hostname. Defaults to localhost.
    * @constructor
    */
-  constructor(port) {
+  constructor(port = DEFAULT_PORT, hostname = DEFAULT_HOSTNAME) {
```

Comment:
these still need to be saved on `this.port` and `this.hostname` (cause of the test failures)